lib/streamlit/bootstrap.py

```diff
@@ -147,7 +148,7 @@ def run(script_path):
     ioloop = tornado.ioloop.IOLoop.current()
 
     # Create and start the server.
-    server = Server(ioloop, script_path, sys.argv)
+    server = Server(ioloop, script_path, sys.argv, command_line)
```

Comment:
Do `argv` and `command_line` serve the same purpose? `command_line` here is purely informative, we don't use it to run the script, but only for the message. it may be worth it to take it a step further and get rid of all these `argv`...

Reply:
Yeah, I mentioned this in another comment. I think you're right: as far as I can tell we can remove all the `argv` now that we can't set command-line arguments via the UI anymore.

Reply:
removed all the way